### DISCUSSION TITLE
bark

### DISCUSSION THREAD

**Discussion by poooooooper** [3kh0/ext-remover#427] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
could you please make point blank (the new one ) work with bark blocker heres its website if you need it bark.us

**Comment by Aka-but-nice** [comment, marked as answer]
There is no link for Bark for chrome, that you can use for Point Blank

**Comment by poooooooper** [comment]
i was wondering if theres a way to make one I could try myself but I literally just started coding so.....idk

**Reply by Aka-but-nice** [reply]
It is not that simple, you would have to find unsafe uses of innerHTML in bark for chrome, as a starter because you just started, I'll give you the crx for it <a href="https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dcros%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D108.0.0.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Djcocgejjjlnfddlhpbecfapicaajdibb%2526uc">here</a> but just to let you know, bark for chrome has no html, which meaning this will not work.
